XamarinCommunityToolkit/Behaviors/ImpliedOrderGridBehavior.shared.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Linq;
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.CommunityToolkit.Behaviors
+{
+	public class ImpliedOrderGridBehavior : BaseBehavior<Grid>
+	{
+		bool[][] usedMatrix;
+		int rowCount;
+		int columnCount;
+
+		public bool ThrowOnLayoutWarning { get; set; }
```

Comment:
Just wonder is there any sense to keep this property?

Reply:
I think so, at least my thought process is when you are working on the layout, you might want to know if there is a layout issues you missed.  Setting that flag just throws exceptions besides logging to the output.